tests/loginActions.test.jsx

```diff
@@ -31,23 +31,44 @@ describe(" sigup actions ", () => {
     const store = mockStore({ item: {} });
     store.dispatch(login(dataObject.data));
     expect(store.getActions()).toEqual([]);
+    store.dispatch(login({errors: "error message"}));
+    expect(store.getActions()).toEqual([]);
 
   });
 
   it("logins in user with login action", () => {
     fetch.mockReject(new Error("fake error message"));
-    const store = mockStore({ item: {} });
-
+    const store = mockStore({ errors: {} });
     store.dispatch(login(dataObject.data));
     expect(store.getActions()).toEqual([]);
+    store.dispatch(login({errors: "error message"}));
+    expect(store.getActions()).toEqual([]);
+
+    expect(alert("error","this is an error message", null, null, "/")).toEqual(undefined);
 
   });
+  it("logins in user with login action", () => {
+    fetchMock.post(loginUrl, {errors: "this error!"}, 200);
+      // .catch(error => (console.log("this>>>", error)));
```

Comment:
Could you remove this commented out code

Reply:
Working on it